src/templates/recipe.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+
+import Layout from '../components/layout';
+import RecipeTimingInformation from '../components/recipe-timing-information';
+import styles from './recipe.module.css';
+
+const renderCreditLink = credit => {
```

Comment:
Mostly personal preference. I doubt this will be reusable anywhere and it's pretty small. IMO this wouldn't make too much sense as a component away from the context of this recipe template.

Reply:
Oh I didn't mean break it out into a separate file necessarily- just a `const CreditLink` in here since it was returning JSX. NBD though.

Reply:
Oh, gotcha. Idk why, but when I think if a component my mind immediately goes to new file. I shouldn't probably break that habbit